shared/reducers/entities.js

```diff
@@ -24,7 +24,7 @@ export default function(state: State = initialState, action: Actions): State {
     case 'entity:replace': {
       const {keyPath, entities} = action.payload
       // $FlowIssue doesn't understand this API
-      return state.mergeIn(keyPath, entities)
+      return state.setIn(keyPath, entities)
```

Comment:
Mmm I don't think so. We want to replace the entities in there. Not replace the entire key path. Maybe we need 2 actions?